firepit/query.py

```diff
@@ -293,7 +305,7 @@ def render(self, placeholder):
         for stage in self.stages:
             text = stage.render(placeholder)
             if isinstance(stage, Table):
-                query = f'FROM "{text}"'
+                query = f'FROM {text}'
```

Comment:
Won't this break for table names that include dashes, e.g.`network-traffic`?

Reply:
You are write, however I wanted to move the table name creation and quoting to the Table class. I have extended the signature to allow for the prefix and an alias to be specified so that table names could be constructed like 
<prefix><name> AS <alias>  could be created
example
"00000"."network-traffic" AS x

I made a similar change to the Join class, to allow for joins like
FROM "00000"."network-traffic" AS x INNER JOIN "00000"."some_var" AS y ON x.id=y.id

This will also require a change to the summary and relations files in Kestrel-lang so I kept them as optional with defaults.

Reply:
Are the aliases required?

Reply:
I removed the aliases and the tests still pass, so it looks like we can leave them out